geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionClear.java

```diff
@@ -401,7 +395,7 @@ protected void handleClearFromDepartedMember(InternalDistributedMember departedM
     }
   }
 
-  class LockForListenerAndClientNotification {
+  static class LockForListenerAndClientNotification {
```

Comment:
Any reason for changing it to static? The PRClear is separate for each PR.

Reply:
This was just a suggestion from the IDE. I'm happy to change it back if it's better to do so.

Reply:
It should not be static.